<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MIDISettings</class>
 <widget class="QDialog" name="MIDISettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="m_listInputPorts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInputPorts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select MIDI Input Ports</string>
   </property>
  </widget>
  <widget class="QListWidget" name="m_listOutputPorts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutputPorts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select MIDI Output Ports</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_outMMCCheckbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Send MMC Using Selected Output Port</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_outMTCCheckbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Send MTC Using Selected Output Port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_inMMCCheckbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Receive MMC Using Selected Input Port</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_inMTCCheckbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Receive MTC Using Selected Input Port</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MIDISettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MIDISettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
